src/styles/utils/_variables.scss

```
@@ -0,0 +1,5 @@
+$day-width: 100px;
+$day-height: 100px;
+$calendar-padding: 10px;
+$color-day-background: #eee;
+$color-day-border: #000;
```

In my opinion namic color variable with color improves code readability:
```suggestion
$color-grey: #eee;
$color-black: #000;
```